specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/managedInstances.json

```diff
@@ -0,0 +1,842 @@
+        },
+        "storageSizeInGB": {
+          "format": "int32",
+          "description": "Storage size in GB. Minimum value: 32. Maximum value: 8192. Increments of 32 GB allowed only.",
+          "type": "integer"
+        },
+        "collation": {
+          "description": "Collation of the managed instance.",
+          "type": "string",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "dnsZone": {
+          "description": "The Dns Zone that the managed instance is in.",
+          "type": "string",
+          "readOnly": true
+        },
+        "dnsZonePartner": {
+          "description": "The resource id of another managed instance whose DNS zone this managed instance will share after creation.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "publicDataEndpointEnabled": {
+          "description": "Whether or not the public data endpoint is enabled.",
+          "type": "boolean"
+        },
+        "sourceManagedInstanceId": {
+          "description": "The resource identifier of the source managed instance associated with create operation of this instance.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "restorePointInTime": {
+          "format": "date-time",
+          "description": "Specifies the point in time (ISO8601 format) of the source database that will be restored to create the new database.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "proxyOverride": {
+          "description": "Connection type used for connecting to the instance.",
+          "enum": [
+            "Proxy",
+            "Redirect",
+            "Default"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ManagedInstanceProxyOverride",
+            "modelAsString": true
+          }
+        },
+        "timezoneId": {
+          "description": "Id of the timezone. Allowed values are timezones supported by Windows.\r\nWindows keeps details on supported timezones, including the id, in registry under\r\nKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Time Zones.\r\nYou can get those registry values via SQL Server by querying SELECT name AS timezone_id FROM sys.time_zone_info.\r\nList of Ids can also be obtained by executing [System.TimeZoneInfo]::GetSystemTimeZones() in PowerShell.\r\nAn example of valid timezone id is \"Pacific Standard Time\" or \"W. Europe Standard Time\".",
+          "type": "string",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "instancePoolId": {
+          "description": "The Id of the instance pool this managed server belongs to.",
+          "type": "string"
+        },
+        "maintenanceWindowSettings": {
```

Comment:
>maintenanceWindowSettings [](start = 9, length = 25)

Please do consider aligning with Microsoft.Maintenance RP for defining maintenance window etc.